my-app/src/Container/chat/chat.js

```diff
@@ -0,0 +1,49 @@
+import react, { useCallback, useMemo } from "react";
+import "./styles/chat.scss";
+import { Interlocutor } from "./interlocutor/interlocutor.js";
+import { MessageMyself } from "./messageMyself/messageMylef.js";
+import { MessageInterlocator } from "./messageInterlocator/messageInterlocator.js";
+import { Input } from "./input/input.js";
+
+export function Chat({ myself, iterlocutor, startChat }) {
```

Comment:
let's discuss variable names

Reply:
corrected variable names to be as simple as possible and describing their meaning